mhs/mhs/common/state/persistance_adapter.py

```diff
@@ -0,0 +1,8 @@
+"""
+This module defines the state adapter interface, used to allow support for multiple state database implementations.
+"""
+
+
+class PersistanceAdapter:
```

Comment:
```suggestion
class PersistenceAdapter:
```
Would also require a renaming the file.